### ISSUE TITLE
Problem with package: crosslingual-coreference

### ISSUE THREAD

**Issue by Sar2580P** [piwheels/packages#394]
### Package name

crosslingual-coreference

### Package version

0.3.1

### PyPI URL

https://pypi.org/project/crosslingual-coreference/

### piwheels URL

https://www.piwheels.org/project/crosslingual-coreference/

### Python version

- [ ] Python 3.7
- [ ] Python 3.9
- [X] Python 3.11

### I am the maintainer

- [ ] Yes

### More information

When install the above package using following command, 
        pip install crosslingual-coreference

Error : 
Collecting checklist<0.0.12,>=0.0.11 (from crosslingual-coreference)
  Using cached checklist-0.0.11.tar.gz (12.1 MB)
  Preparing metadata (setup.py) ... error
  error: subprocess-exited-with-error

  × python setup.py egg_info did not run successfully.
  │ exit code: 1
  ╰─> [49 lines of output]

note: This error originates from a subprocess, and is likely not a problem with pip.
error: metadata-generation-failed

× Encountered error while generating package metadata.
╰─> See above for output.

note: This is an issue with the package mentioned above, not pip.

**Comment by bennuttall** [comment]
**Can I just check you are aware this is the issue tracker for a Python package index specifically for Raspberry Pi? https://www.piwheels.org/**

It's been quite difficult to track the issue down. I found a missing version of spacy which I've now resolved, but I still get a dependency conflict error:

```
ERROR: Cannot install crosslingual-coreference because these package versions have conflicting dependencies.

The conflict is caused by:
    allennlp 2.9.3 depends on torch<1.12.0 and >=1.6.0
    allennlp 2.9.2 depends on torch<1.12.0 and >=1.6.0
    allennlp 2.9.1 depends on torch<1.11.0 and >=1.6.0
    allennlp 2.9.0 depends on torch<1.11.0 and >=1.6.0

To fix this you could try to:
1. loosen the range of package versions you've specified
2. remove package versions to allow pip attempt to solve the dependency conflict

ERROR: ResolutionImpossible: for help visit https://pip.pypa.io/en/latest/topics/dependency-resolution/#dealing-with-dependency-conflicts
```

I don't think it's possible to install this on a Raspberry Pi because we don't have torch: #213

**Comment by MichaIng** [comment]
Indeed, this package should be probably removed from piwheels, same as `allennlp`: `crosslingual-coreference` depends on `allennlp` which depends on `torch`. And `torch` has no sdist and wheels are provided for `x86_64` and `aarch64` only: https://pypi.org/project/torch/#files

Hence `crosslingual-coreference` and `allennlp` cannot be installed on any 32-bit ARM system, and building these wheels just unnecessarily consumes builder time.
